### PR TITLE
Add maanas Bio

### PR DESCRIPTION
**Pull request by maanasnandu**
Added Maanas's Bio in the team page

**Before:**
Does not contain Maanas's Bio in the team page.

**After:**
Contains Maanas's Bio in the team page without a picture.

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -254,6 +254,34 @@ <h6 class="card-subtitle pt-1 pb-3" style="color: #EF4035">Treasurer</h6>
                         </div>
                      </div>
                   </div>
+
+                  <div class="card my-2">
+                     <img class="card-img-top p-2" src="" alt="Manisha Kusuma">
```
Comment (hassanabral) [inline team.html]: Add your own image (make sure you change the "alt" text). Also, make sure that the image is compressed.

File: team.html [hunk 2]
```diff
@@ -254,6 +254,34 @@ <h6 class="card-subtitle pt-1 pb-3" style="color: #EF4035">Treasurer</h6>
...
+                     <div class="card-body">
+                        <h4 class="card-title officer-name">Maanas Muddam</h4>
+                        <h6 class="card-subtitle pt-1 pb-3" style="color: #EF4035">webmaster</h6>
```
Comment (hassanabral) [inline team.html]: Change this to capital case.

File: team.html [hunk 3]
```diff
@@ -254,6 +254,34 @@ <h6 class="card-subtitle pt-1 pb-3" style="color: #EF4035">Treasurer</h6>
...
+                        <p class="card-text">Maanas is a junior studying CS with a minor in Mathematics. He joined WebDVT after being exposed to front-end technologies during her internship and wanted to broaden her understanding. On her spare time she likes to hike and travel.
+                        </p>
+                        <hr>
+                        <div class="row footer-social">
+                           <div class="col-lg-12">
+                              <ul class="list-inline">
+                                 <li class="list-inline-item">
+                                    <a target="_blank" href="https://www.linkedin.com/in/manisha-ku/">
```
Comment (hassanabral) [inline team.html]: Add your social media links

**Review by hassanabral — COMMENTED** [review]
Good job, keep working!

**Review by maanasnandu — COMMENTED** [review]
Added profile image bio and social links to Maanas bio in team page
**Before:**
Did not contain bio, social links and profile image.

**After:**
Contain bio, social links and profile image of Maanas

**Review by hassanabral — CHANGES_REQUESTED** [review]
Everything looks good to me except one minor thing.

Since we have three columns in the second row now, could you remove the `mx-md-5` and `px-md-5` class names from this line?
https://github.com/webdvt/webdvt/blob/maanasBio/team.html#L230

**Comment by hassanabral** [comment]
When you make new commits to a PR, instead of making a new comment, just update the `PR description` at the top. Also, make sure to remove the `WIP` from the PR title when you are satisfied with the PR and is ready to merge.

Otherwise, great work! 👍

**Review by maanasnandu — COMMENTED** [review]
Removed mx-md-5 and px-md-5 class names from card deck class
**Before:**
Contained mx-md-5 and px-md-5 classes for card desk class at line 230 which is causing the bio cards to 
slightly shrink.

**After:**
Removed the mx-md-5 and px-md-5 class names  from card desk class.

**Review by hassanabral — APPROVED** [review]
💯

**Pull request merged**